docs/about/release-notes.md

```diff
@@ -1724,26 +1724,26 @@ the `extra_css` or `extra_javascript` config settings going forward.
 * File permissions are no longer copied during build (#1292)
 * Support query and fragment strings in `edit_uri` (#1224 & #1273)
 
-## Version 0.16.3 (2017-04-04)
+## [Version 0.16.3] (2017-04-04)
 
 * Fix error raised by autoscrolling in the readthedocs theme (#1177)
 * Fix a few documentation typos (#1181 & #1185)
 * Fix a regression to livereload server introduced in 0.16.2 (#1174)
 
-## Version 0.16.2 (2017-03-13)
+## [Version 0.16.2] (2017-03-13)
 
 * System root (`/`) is not a valid path for site_dir or docs_dir (#1161)
 * Refactor readthedocs theme navigation (#1155 & #1156)
 * Add support to dev server to serve custom error pages (#1040)
 * Ensure nav.homepage.url is not blank on error pages (#1131)
 * Increase livereload dependency to 2.5.1 (#1106)
 
-## Version 0.16.1 (2016-12-22)
+## [Version 0.16.1] (2016-12-22)
 
 * Ensure scrollspy behavior does not affect nav bar (#1094)
 * Only "load" a theme when it is explicitly requested by the user (#1105)
 
-## Version 0.16 (2016-11-04)
+## [Version 0.16] (2016-11-04)
```

Comment:
Headers like this don't reflect the real ones, may need to update them manually.

We can also set up a workflow like https://github.com/GradleUp/shadow/pull/1373 to check links and update them collectively.